src/dorsal.js

```diff
@@ -160,7 +160,11 @@ DorsalCore.prototype._runPlugin = function(el, pluginName) {
  * @return {Array} registered plugin names
  */
 DorsalCore.prototype.registeredPlugins = function() {
-    return Object.keys(this.plugins);
```

Comment:
I don't remember if this is a required thing. maybe @parris or @mralex remembers why.